src/DssSpell.sol

```diff
@@ -16,135 +16,265 @@
+            DssExecLib.authorize(addr, NEW_MCD_ESM);
+        }
+
+        DssExecLib.setChangelogAddress(MCD_ESM_NAME, NEW_MCD_ESM);
+        DssExecLib.setChangelogVersion("1.10.0");
+
+        //////////////////////////////////////////////////////////
+        // Delegate Compensation January Distribution
+        // https://forum.makerdao.com/t/recognized-delegate-compensation-breakdown-january-2022/13001
+
+        DssExecLib.sendPaymentFromSurplusBuffer(FLIP_FLOP_FLAP_WALLET,  12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(FEEDBLACK_LOOPS_WALLET, 12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(SCHUPPI_WALLET,         12_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(MAKERMAN_WALLET,         8_620);
+        DssExecLib.sendPaymentFromSurplusBuffer(MONETSUPPLY_WALLET,      4_807);
+        DssExecLib.sendPaymentFromSurplusBuffer(ACRE_INVEST_WALLET,      3_795);
+        DssExecLib.sendPaymentFromSurplusBuffer(JUSTIN_CASE_WALLET,        889);
+        DssExecLib.sendPaymentFromSurplusBuffer(GFX_LABS_WALLET,           641);
+
+
+        //////////////////////////////////////////////////////////
+        // Repair Dai Streams
+        // https://forum.makerdao.com/t/correction-to-last-weeks-executive/13022
+
+        // MIP40c3-SP47: Core Unit Budget (SNE-001) - Phase II StarkNet Fast Withdrawal and Wormhole
+        // https://mips.makerdao.com/mips/details/MIP40c3SP47
+        uint256 _sneId = 24;
+        // Send first month payment minus accrued amount
+        uint256 snePayment = sub(42_917 * WAD, VEST.accrued(_sneId));
+        VatAbstract(MCD_VAT).suck(MCD_VOW, address(this), snePayment * RAY);  // WAD * RAY == RAD
+        DaiJoinAbstract(MCD_JOIN_DAI).exit(SNE_001_WALLET, snePayment);
+        // Cancel
+        VEST.unrestrict(_sneId);
+        VEST.vest(_sneId); // Pay unpaid stream amount
+        VEST.yank(_sneId);
+        // Stream payments for months 2-6
+        VEST.restrict(
+            VEST.create(SNE_001_WALLET,     214_583 * WAD, MAR_01_2022, JUL_31_2022 - MAR_01_2022,            0, address(0))
+        );
+        // Total of payments = 257_500
+
+        // MIP40c3-SP46: Adding Strategic Finance Core Unit Budget (SF-001)
+        // https://mips.makerdao.com/mips/details/MIP40c3SP46
+        uint256 _sfId = 26;
+        // Send first month payment minus accrued amount
+        uint256 sfPayment = sub(82_417 * WAD, VEST.accrued(_sfId));
+        VatAbstract(MCD_VAT).suck(MCD_VOW, address(this), sfPayment * RAY);
+        DaiJoinAbstract(MCD_JOIN_DAI).exit(SF_001_WALLET, sfPayment);
+        // Cancel stream
+        VEST.unrestrict(_sfId);
+        VEST.vest(_sfId); // Pay unpaid stream amount
+        VEST.yank(_sfId);
+        // Stream payments for months 2-6
+        VEST.restrict(
+            VEST.create(SF_001_WALLET,      412_085 * WAD, MAR_01_2022, JUL_31_2022 - MAR_01_2022,            0, address(0))
+        );
+        // Total of payments = 494_502
 
```

Comment:
nit: extra line

Reply:
thanks already deployed so will skip this.